src/components/Form.jsx

```diff
@@ -36,7 +36,7 @@ const Form = () => {
 
   return (
     <section className={styles.addbook}>
-      <div className={styles['addbook__wrapper']}>
+      <div className={styles.addbook__wrapper}>
         <h2 className={styles.heading}>Add new book</h2>
         <form className={styles.form} onSubmit={handleSubmit}>
```

Comment:
- [ Optional ] I think it would be nice if you make sure that the `title` and `author` input fields are validated to only accept valid input. I noticed that it's possible for a user to submit an empty book which should not be the case, please see the screenshot below and rectify this.

## Error when adding a book
![Screenshot from 2023-09-30 09-57-04](https://github.com/fmanimashaun/bookstore-cms/assets/16392046/f98d17eb-5e9a-4aea-a3f9-57cbf29f8ef3)

Reply:
I will look into it. thanks